src/enums.js

```diff
@@ -74,4 +75,4 @@ exports.ThermostatRoomSensor = ThermostatRoomSensor
 //SCALE_S = ['WL_SC2']
 //SCALE_X = ['WL_SC22135']
 //WATCH = ['RA.WP1', 'RY.WA1']
-//Wrist = ['RY.HP1']
```

Comment:
What is this change at the end of the file? Probably a newline error, please clean that up then we can merge.

Reply:
@carTloyal123 this is actually POSIX standard to append a new line in the end of file. many editors/IDEA will do it automatically. 

please let me know if you still want me to remove the newline or not


https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline